Jenkinsfile

```diff
@@ -62,7 +62,7 @@ lines.each {line ->
                 unstash 'excludes.txt'
                 unstash 'pct'
                 unstash "megawar-$line"
-                withEnv(["PLUGINS=$plugin", "LINE=$line", 'EXTRA_MAVEN_PROPERTIES=surefire.rerunFailingTestsCount=4']) {
+                withEnv(["PLUGINS=$plugin", "LINE=$line", 'EXTRA_MAVEN_PROPERTIES=maven.test.failure.ignore=true:surefire.rerunFailingTestsCount=4']) {
```

Comment:
Consistent with the `Jenkinsfile` for core and plugins and needed because Maven would otherwise now exit nonzero on failing tests which would interfere with the `junit` step in the `Jenkinsfile`.